code/modules/research/designs/weapons.dm

```diff
@@ -212,7 +212,7 @@
 	id = "smg"
 	req_tech = list("combat" = 4, "materials" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 8000, MAT_SILVER = 2000, MAT_DIAMOND = 1000)
+	materials = list(MAT_IRON = 10000, MAT_SILVER = 2000)
```

Comment:
Because I was planning to make it slightly weaker to compensate for its price. As it is now, it's a 4-shot crit.